src/css/custom.css

```diff
@@ -24,10 +24,22 @@
       url("/static/fonts/Torus-SemiBold.otf") format("TorusSemiBold");
     font-weight: 200 900;
     font-stretch: 75% 125%;
+    unicode-range: U+0000-00FF, U+0100-017F, U+0180-024F;
   }
   
+  @font-face {
+    font-family: "HarmonyOS Sans";
+    src: url("/static/fonts/HarmonyOS Sans/HarmonyOS_Sans_Black.ttf") format("truetype"), /* Add other weights/styles as needed */
+    url("/static/fonts/HarmonyOS Sans/HarmonyOS_Sans_Bold.ttf") format("truetype"),
+    url("/static/fonts/HarmonyOS Sans/HarmonyOS_Sans_Light.ttf") format("truetype"),
+    url("/static/fonts/HarmonyOS Sans/HarmonyOS_Sans_Medium.ttf") format("truetype"),
+    url("/static/fonts/HarmonyOS Sans/HarmonyOS_Sans_Regular.ttf") format("truetype"),
+    url("/static/fonts/HarmonyOS Sans/HarmonyOS_Sans_Thin.ttf") format("truetype");
+    unicode-range: U+4E00-9FFF, U+3400-4DBF, U+20000-2A6DF, U+2A700-2B73F, U+2B740-2B81F, U+2B820-2CEAF;
+  }
+
```

Comment:
Use `HarmonyOS Sans` for CJK characters display.

Reply:
Can consider adding 3000-303F to this range, which consists of some common CJK symbols and punctuations.

![table](https://github.com/user-attachments/assets/94a99480-b377-46ca-a6b5-a06c27a3a54d)

Reply:
in commit [65d95f0](https://github.com/offfc/offfc.github.io/pull/5/commits/65d95f092de8f20cb7f18ab5c9d47e15d754d17b).

Reply:
After a time of research, I tried to make the family definition at the root level of the stylesheet, and somehow it worked:

![mixed fonts](https://github.com/user-attachments/assets/d0ba53e8-0a1f-4ca3-8441-670f35bb43d8)

How strange.

Reply:
Great.

Push that commit onto this branch and we will merge this PR.